BlogTemplate/Pages/New.cshtml

```diff
@@ -13,15 +12,23 @@
 
 <h2>Write a New Post</h2>
 <form method="POST">
-    <p><h3><label for="Title">Title</></label></h3></p>
-    <p><input type="text" name="Title" asp-for="Post.Title" /></p>
-    <p><h4><label for="Body">Body</label></h4></p>
-    <p><textarea name="Body" asp-for="Post.Body"></textarea></p>
-    <p><label for="Excerpt">Excerpt (Optional)</label></p>
-    <p><textarea name="Excerpt" asp-for="Post.Excerpt"></textarea></p>
-    <p><label for="Tags">Tags (separated by comma)</label></p>
-    @*<p><input type="text" name="Tags" asp-for="Post.Tags"/></p>*@
-    <p><input type="text" name="Tags" /></p>
+    @Html.ValidationSummary()
```

Comment:
We should keep using tag helpers here instead of HTML helpers.  See https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/intro for a comparison of the two.